README.md

```diff
@@ -57,6 +57,41 @@ This experiment is enabled by default and sessions between two Pat nodes (or oth
 
 For more information, see <https://github.com/la5nta/wl2k-go#gzip-experiment>.
 
+
+## Docker
+
+### Building
+
+To build Pat as a Docker image run the following:
+
+```
+docker build . -t latest
+```
+
+### Running the Docker image
+
+Make a directory called `docker/config` and creat a `config.json` file in it. This is the configuration file the Docker container will mount. Make sure the `http_addr` is set to `:8080` in `config.json` otherwise the web interface won't function inside the container.
+
+```
+# Build folder structure.
```

Comment:
Is this really needed? We automatically create these directories if they don't exist.

Reply:
AFAIK you need to build the local directories on your system before Docker can use them.